### PR TITLE
Fix drain creation authorization

### LINKED ISSUE #211: Unable to create an ElasticSearch drain
When I try to create an ElasticSearch drain, I'll get this error while I'm providing credentials:

```
$ clever drain create --verbose --alias my-app ElasticSearch http://elasticsearchinstance-elasticsearch.services.clever-cloud.com --username john --password changeme
[DEBUG] Load configuration from /Users/john/.config/clever-cloud
[DEBUG] Loading app configuration from /Users/john/src/my-app/.clever.json
[DEBUG] Registering drain for app_fc5b6d60-2f66-11e8-8a3b-43a11d499aa2
[ERROR] Credentials are: optional for HTTP, mandatory for ElasticSearch and TCPSyslog/UDPSyslog don't need them.
```

### PR DESCRIPTION
**Pull request by seblm**
Closes #211

### CODE REVIEW

File: spec/drain.spec.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const expect = require('chai').expect;
+
+const drain = require('../src/models/drain.js');
+
+describe('drain', function () {
+
+  it('authorize ElasticSearch with credentials', () => {
```
[the remark this answers is not in the document]
Comment (seblm) [reply spec/drain.spec.js]: For sure. Done.

File: spec/drain.spec.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    const isAuthorized = drain.authorizeDrainCreation('ElasticSearch', {username: 'john', password: 'changeme'});
+    expect(isAuthorized).to.be.true;
+  });
+
+  it('not authorize ElasticSearch without credentials', () => {
```
[the remark this answers is not in the document]
Comment (seblm) [reply spec/drain.spec.js]: Done as well.

**Comment by seblm** [comment]
Please let me known when it is ok to merge : I'll rebase with autosquash.

**Comment by seblm** [comment]
ok commits are rebased

**Comment by KannarFr** [comment]
Thanks.

**Comment by hsablonniere** [comment]
@seblm Your fix is in the latest beta. Please see the README if you want to install try the beta.

**Pull request merged**